boomer/boomer.go

```diff
@@ -127,6 +131,9 @@ func (b *Boomer) runWorker(n int) {
 		TLSHandshakeTimeout: time.Duration(b.Timeout) * time.Millisecond,
 		Proxy:               http.ProxyURL(b.ProxyAddr),
 	}
+	if b.H2 {
+		http2.ConfigureTransport(tr)
```

Comment:
Why don't we just do the recommended way and set the TLSNextProto? I don't know if it is already disable and I don't honestly want to care about those edge cases :)